monero-outreach-docs/translations/es/storyboard/wallet_quickstart_es.md

```diff
@@ -0,0 +1,56 @@
+
+Al escoger entre estos, ten en cuenta que la mayoría tiene una opción extra en la cual — siendo la excepción The Edge Wallet — pueden ser configurados para conectarse ya sea a un servidor remoto de terceros o a un servidor que ejecutes con un nodo. Esto se ilustra en la tabla.
+
+## Monederos Oficiales
+
++ [getmonero.org/es/downloads](https://www.getmonero.org/es/downloads)  
+Si deseas ejecutar tu propio nodo completo, una buena elección es el monedero Monero GUI o CLI oficial. El primero provee una interfaz gráfica de usuario, y el último una interfaz textual.
+
+## Monederos Móviles
+
++ [cakewallet.io](https://cakewallet.io/)  
+Cake Wallet, un monedero ligero de protección de claves-de-visualización, está disponible para dispositivos iOS.
+
++ [monerujo.io](https://www.monerujo.io/)  
+Monerujo, un monedero ligero de protección de claves-de-visualización, está disponible para dispositivos Android.
+
++ [edge.app](https://edge.app/)  
+Edge Wallet está disponible para ambos Android y iOS. Es un monedero ligero de compartir claves de visualización.
+
+## Monederos Web *(Optimizados para Conveniencia)*
+
++ [mymonero.com](https://mymonero.com/)  
+MyMonero es un monedero ligero, basado en navegador. También está disponible como una app de escritorio y app de iOS.
+
+## Monederos Hardware *(Optimizados para Seguridad)*
+
++ [ledger.com](https://shop.ledger.com/?r=92d74dc2847a)  
+Ledger es un dispositivo altamente seguro que guarda tu semilla mnemónica. No aplica.
+
++ [trezor.io](https://trezor.io/)  
+Trezor es una alternativo de dispositivo altamente seguro que guarda tu semilla mnemónica.
+
+Para un monedero hardware que provea seguridad extra en almacenar y transmitir Monero incluso si el dispositivo primario se ve comprometido, ambos Ledger, [ledger.com](https://shop.ledger.com/?r=92d74dc2847a), y Trezor, [trezor.io](https://trezor.io/), son una buena consideración. (Nótese que Ledger trabaja ahora con monederos, mientras que el soporte para monedero en Trezor está en progreso).
```

Comment:
(Nótese que Ledger trabaja ahora con monederos, mientras que el soporte para **el** monedero en Trezor está en progreso).

I think it's missing "el".